### PR TITLE
Update strings.po uk_ua

### PR DESCRIPTION
**Pull request by dm1tz**

### CODE REVIEW

File: language/resource.language.uk_ua/strings.po [hunk 1]
```diff
@@ -750,12 +750,12 @@ msgstr "Перед видаленням іконки опцій, перекон
 #: /1080i/Includes_Defaults.xml
 msgctxt "#31156"
 msgid "You have $INFO[Window(Home).Property(Music.ArtistsCount),[COLOR=main_fg_90][B],[/B][/COLOR]] artists$COMMA $INFO[Window(Home).Property(Music.AlbumsCount),[COLOR=main_fg_90][B],[/B][/COLOR]] albums and $INFO[Window(Home).Property(Music.SongsCount),[COLOR=main_fg_90][B],[/B][/COLOR]] songs in your library."
-msgstr ""
+msgid "Каталог налічує $INFO[Window(Home).Property(Music.ArtistsCount),[COLOR=main_fg_90][B],[/B][/COLOR]] виконавців$COMMA $INFO[Window(Home).Property(Music.AlbumsCount),[COLOR=main_fg_90][B],[/B][/COLOR]] альбомів та $INFO[Window(Home).Property(Music.SongsCount),[COLOR=main_fg_90][B],[/B][/COLOR]] пісень."
```
Comment (icarok99) [inline language/resource.language.uk_ua/strings.po:753]: Needs to be edited, you changed the msgstr to the msgid.

File: language/resource.language.uk_ua/strings.po [hunk 2]
```diff
@@ -750,12 +750,12 @@ msgstr "Перед видаленням іконки опцій, перекон
...
 
 #: /1080i/Includes_Defaults.xml
 msgctxt "#31157"
 msgid "The album $INFO[Window(Home).Property(LatestAlbum.1.Title),[COLOR=main_fg_90][B],[/B][/COLOR]] by $INFO[Window(Home).Property(LatestAlbum.1.Artist),[COLOR=main_fg_90][B],[/B][/COLOR]] was most recently added."
-msgstr ""
+msgid "Альбом $INFO[Window(Home).Property(LatestAlbum.1.Title),[COLOR=main_fg_90][B],[/B][/COLOR]] виконавця $INFO[Window(Home).Property(LatestAlbum.1.Artist),[COLOR=main_fg_90][B],[/B][/COLOR]] було додано останнім."
```
Comment (icarok99) [inline language/resource.language.uk_ua/strings.po:758]: This one too

**Review by icarok99 — COMMENTED** [review]
Needs to be edited, you changed the msgstr to the msgid

File: language/resource.language.uk_ua/strings.po (hunk 1 above)
Comment (icarok99) [inline language/resource.language.uk_ua/strings.po]: 👀

File: language/resource.language.uk_ua/strings.po (hunk 2 above)
Comment (icarok99) [inline language/resource.language.uk_ua/strings.po]: 👀

**Review by icarok99 — CHANGES_REQUESTED** [review]

**Comment by dm1tz** [comment]
Whoops, my bad. Thanks @icarok99!

**Review by icarok99 — APPROVED** [review]

**Pull request merged**